src/ott/tools/plot.py

```diff
@@ -120,13 +122,26 @@ def __call__(self, ot: Transport) -> List[plt.Artist]:
     cmap = plt.get_cmap(self._cmap)
     self._lines = []
     for start, end, strength in lines:
+
+      # Make the transparency of the lines depend on the strength of the coupling.
+      alpha = 0.7
```

Comment:
I'd consider these changes:
- move the computation of normalization out of the loop (it's costly)
- allow passing `alpha: float = 0.7` in the `__init__` instead of it being hard-coded
- assign `ot.matrix` to a variable (it it being possibly re-computed)
- also to save some computation

Reply:
Adressed these changes

- I put the max and min out of the loop, but I kept a quick normalization step in the loop (scalar multiplication only)
- I put alpha in the init
- I assigned ot.matrix to a variable